src/config.cpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2017 The Bitcoin developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
```

Comment:
wonder if we could put the content of this file in `src/globals.cpp` in a future PR?

Reply:
Good question...I think it's intent is correct and helps to pass the correct config based on what environment we're running in rather than a global var which we need regardless of environment